src/data.ts

```diff
@@ -0,0 +1,31 @@
+const myInformation = {
+    contactInfo: {
+        fullName: "Manuel Alessandro Concoba Núñez",
+        emailAddress: "mconcoba20@gmail.com",
+    },
+    github: {
+        profileURL: "https://github.com/MConcoba",
+        username: "MConcoba",
+    },
+    credentials: {
+        password: "",
+    },
+    personalInfo: {
+        questions: [
+            {
+                question: "If I was a Sr. Programmer, I would like to build:",
+                answer: "Functional programming solutions for people who lack the tools or skills to achieve their goal, as long as it is for the common good.",
+            },
+            {
+                question:"Por favor indica el URL que me lleva a la línea de código de la definición de React.useEffect",
+                answer: "https://github.com/facebook/react/blob/655affa302437208e6f03c9ca6d170ea1707ace3/packages/react-reconciler/src/ReactFiberHooks.new.js#L1878",
+            },
+            {
+                question: "code is poetry, because:",
+                answer: "Poetry is art, art is defined as a way to convey feelings creating something that is unique and original.",
+            },
+        ],
+    },
+}
+
+export = myInformation;
```

Comment:
Este syntax no es muy claro, es mejor si hacemos: 

```suggestion
export default myInformation;
```